setup/webmail.sh

```diff
@@ -209,6 +210,16 @@ php$PHP_VER ${RCM_DIR}/bin/updatedb.sh --dir ${RCM_DIR}/SQL --package roundcube
 chown www-data:www-data $STORAGE_ROOT/mail/roundcube/roundcube.sqlite
 chmod 664 $STORAGE_ROOT/mail/roundcube/roundcube.sqlite
 
+# Patch the Roundcube code to eliminate an issue that causes postfix to reject our sqlite
+# user database (see https://github.com/mail-in-a-box/mailinabox/issues/2185)
+sed -i.miabold 's/^[^#]\+.\+PRAGMA journal_mode = WAL.\+$/#&/' \
+/usr/local/lib/roundcubemail/program/lib/Roundcube/db/sqlite.php
+
+# Because Roundcube wants to set the PRAGMA we just deleted from the source, we apply it here
+# to the roundcube database (see https://github.com/roundcube/roundcubemail/issues/8035)
+# Database should exist, created by migration script
+sqlite3 $STORAGE_ROOT/mail/roundcube/roundcube.sqlite 'PRAGMA journal_mode=WAL;'
```

Comment:
Might this need to be set to DELETE or TRUNCATE instead if we are to not use WAL anymore?

Reply:
I don't think so. This specific database is owned by roundcube and not used by postfix. Because roundcube would like to use WAL, and we just removed it from the source, we set it here explicitly.

A possible addition is setting the DELETE pragma for the MiaB database users.sqlite. Just to be sure it is set to a value that postfix understands.

Reply:
I think this may break new installations. The roundcube.sqlite file isn't created by the migration script. It checks if the file exists before mangling it (and only does this once in migration_12).

This also emits "wal" to the console so it would be nice to silence it with e.g. `> /dev/null` (if it's to STDOUT and not STDERR).

Reply:
Line 207 states "Run Roundcube database migration script (database is created if it does not exist)", so I kinda depend on that. There's also the chmod and chown commands that are applied to the database. Perhaps the comment about migration should mention the updatedb script? I'll take a closer look. 

I will also look into silencing the command.

Reply:
Ahha! I'm sorry. I thought you meant the Mail-in-a-Box migration system (migrate.sh). I didn't look a few lines above. Yes ok then we are all good.